lib/utils.pm

```diff
@@ -477,16 +482,28 @@ sub reboot_gnome {
     workaround_type_encrypted_passphrase;
 }
 
+=head2 poweroff
+
+    poweroff;
+
+Shutdown system from root console.
+=cut
+sub poweroff {
+    reboot('poweroff');
+}
+
 =head2 reboot
 
     reboot;
 
 Reboot system from root console.
 =cut
 sub reboot {
+    my ($action) = @_;
+    $action //= 'reboot';
```

Comment:
This and the "reboot(poweroff)" sounds like you want a util function "power_action" with commands argument "reboot" or "poweroff". Would that sound more useful?